### PR TITLE
Update unstructured-data-upload.yaml

### PR DESCRIPTION
**Pull request by tbagnallMATM**
@drsmit22 @issmith1 
Hi Dan & Irene, just to start the convo...
I'm not a full-fledged programmer like you two are, this branch is my best attempt at righting some of the wrongs in the existing unstructured data upload yaml. The changes include "breaking" and "non-breaking" ones. 

Dan, I did my best to guess at how you might implement the custom metadata field. I'm sure it doesn't get the Liddell seal of approval.

### CODE REVIEW

File: datacollection/openapi/ati/unstructured-data-upload.yaml [hunk 1]
```diff
@@ -69,7 +69,7 @@ components:
         properties:
           document_id:
             description: >-
-              An ATI created UUID.
+              A UUID as a unique identifier to the postflight data file. Provided by the person performing for the upload (i.e., the "uploader").
```
Comment (issmith1) [inline datacollection/openapi/ati/unstructured-data-upload.yaml:72]: rm 'A UUID as a"

File: datacollection/openapi/ati/unstructured-data-upload.yaml [hunk 2]
```diff
@@ -157,8 +160,16 @@ components:
             type: string
             maxLength: 254
             format: ".+@.+"
-
-
+          custom_field_names:
+            description: An optional field for capturing specific metadata for a data source. For example, weather data files require the specification of station name and point location. This array captures the field names. Content here requires content in the "custom_field_values" field.
```
Comment (issmith1) [inline datacollection/openapi/ati/unstructured-data-upload.yaml]: chg 'An optional field for capturing' to something like 'captures specific...

we want to state its datatype in the type: field

**Review by issmith1 — APPROVED** [review]

File: datacollection/openapi/ati/unstructured-data-upload.yaml (hunk 2 above)
Comment (decet) [reply datacollection/openapi/ati/unstructured-data-upload.yaml]: Like I mention in the email, here's the JSON Schema for the custom data fields.

               {                                                               
                  "type": "array",                                              
                  "minItems": 1,                                                
                  "items": {                                                    
                    "type": "object",                                           
                    "required": [                                               
                      "field_name"                                              
                    ],                                                          
                    "properties": {                                             
                      "field_name": {                                           
                        "type": "string",                                       
                        "minLength": 1,                                         
                        "maxLength": 1000                                       
                      },                                                        
                      "field_value": {                                          
                        "type": "string",                                       
                        "minLength": 0,                                         
                        "maxLength": 1000                                       
                      }                                                         
                    },                                                          
                    "additionalProperties": false                               
                  }                                                             
                }

Comment (decet) [reply datacollection/openapi/ati/unstructured-data-upload.yaml]: I've added my changes to custom_fields to this branch, and Tim's original custom_fields is now labelled "outdated", and there is a single line showing the new commit at the bottom just elow this comment, labelled "af51bb9"

**Pull request merged**